eras/alonzo/test-suite/test/Test/Cardano/Ledger/Alonzo/Serialisation/CDDL.hs

```diff
@@ -33,6 +34,7 @@ tests n = withResource combinedCDDL (const (pure ())) $ \cddl ->
     , cddlTest @(PParamsUpdate Alonzo) (eraProtVerHigh @Alonzo) n "protocol_param_update"
     , cddlAnnotatorTest @(Redeemers Alonzo) (eraProtVerHigh @Alonzo) n "[* redeemer]"
     , cddlAnnotatorTest @(Tx Alonzo) (eraProtVerHigh @Alonzo) n "transaction"
+    , cddlTest @CostModels (eraProtVerHigh @Alonzo) n "costmdls"
```

Comment:
that correct, it would not fail before or after this change. and in fact, cost models can end up in `PParamsUpdate`, so it is not strictly necessary. but I thought it would be nice to add them for easier future-proofing and debugging.

Reply:
Makes sense, thanks for the explanation!